phases/phase-2/README.md

```diff
@@ -0,0 +1,41 @@
+# Phase 2 
+The second phase of the Primer program is based on the `morpheus-2000` testnet. During this phase you will have the chance to become a validator and/or learn more about the replacements of Phase 1's likes: reactions! :tada:
```

Comment:
The second phase of the Primer program is based on the `morpheus-2000` testnet. During this phase you will have the chance to become a validator and/or learn more about the replacements of Phase 1's likes: reactions! :tada:
->
The second phase of the Primer program is based on the `morpheus-2000` testnet. During this phase you will learn how to become a validator. You will also use reactions! :tada:, which has replaced the "like" action in Phase 1.